src/components/modeler/Modeler.vue

```diff
@@ -665,6 +666,10 @@ export default {
       return shape.component != null;
     },
     setShapeStacking(shape) {
+      if (this.isRendering) {
+        return;
+      }
+
```

Comment:
Multiple calls to `toFront` (which is what `ensureShapeIsNotCovered` does) seemed to be called many times when loading the large process, and appeared to create a significant amount of CPU/rendering work.

While this didn't seem to cause any overlapping issues on initial load, it is still required to prevent overlapping after the process has loaded.